Bottom: e916777fa933aad43d527bb487a4b878c8dd0a07
Top:    5211f934e59b9c5cd0798cec867d93c1f891f0f2
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-08 19:46:36 +1300

Refresh of extended-to-stream

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 9e31a3d..8a9ce01 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -1144,7 +1144,7 @@ class ExtendedToStreamDecorator(CopyStreamResult, StreamSummary, TestControl):
         if value:
             if len(self.targets) == 2:
                 return
-            self.targets.append(StreamFailfast(self.stop))
+            self.targets.append(StreamFailFast(self.stop))
         else:
             del self.targets[1:]
     failfast = property(_get_failfast, _set_failfast)
